cypress/e2e/catalog/giftCards/exportGiftCards.js

```diff
@@ -90,49 +83,45 @@ describe("As an admin I want to export gift card", () => {
     "should be able to export several gift cards to xlsx file. TC: SALEOR_1014",
     { tags: ["@giftCard", "@allEnv", "@stable"] },
     () => {
-      const giftCard01 = `${startsWith}${faker.datatype.number()}`;
-      const giftCard02 = `${startsWith}${faker.datatype.number()}`;
+      const firstGiftCardTag = faker.datatype.number();
+      const secondGiftCardTag = faker.datatype.number();
       const exportId = `${faker.datatype.number()}`;
-      let giftCard01hash;
-      let giftCard02hash;
+      let firstGiftCardId;
+      let secondGiftCardId;
+      let firstGiftCardCode;
+      let secondGiftCardCode;
 
       updatePlugin(
         "mirumee.notifications.admin_email",
         "csv_export_success_subject",
         `Your exported {{ data_type }} data #${exportId} is ready`,
       );
       createGiftCard({
-        tag: giftCard01,
+        tag: firstGiftCardTag,
         amount: 5,
         currency: "THB",
       })
-        .then(hash => {
-          giftCard01hash = hash.id;
+        .then(firstGiftCard => {
+          firstGiftCardId = firstGiftCard.id;
+          firstGiftCardCode = firstGiftCard.code;
           createGiftCard({
-            tag: giftCard02,
+            tag: secondGiftCardTag,
             amount: 10,
             currency: "THB",
           });
         })
-        .then(hash2 => {
-          giftCard02hash = hash2.id;
-          enterAndSelectGiftCards([giftCard01hash, giftCard02hash]);
-          cy
-            .get(ASSIGN_ELEMENTS_SELECTORS.checkbox)
-            .first()
-            .check()
-            .should("be.checked")
-            .get(GIFT_CARD_LIST.selectedAmount)
-            .contains("Selected 2 items")
-            .should("be.visible")
-            .get(BUTTON_SELECTORS.showMoreButton)
-            .click({ force: true })
-            .get(GIFT_CARD_SHOW_MORE.exportCodesMenu)
-            .click()
-            .get(GIFT_CARD_SHOW_MORE.exportAsRadioBtn.xlsx)
-            .click()
-            .get(BUTTON_SELECTORS.submit)
-            .click().confirmationMessageShouldDisappear;
+        .then(secondGiftCard => {
+          secondGiftCardId = secondGiftCard.id;
+          secondGiftCardCode = secondGiftCard.code;
+          cy.visit(
+            `${urlList.giftCards}?asc=true&sort=usedBy&tag%5B0%5D=${firstGiftCardTag}&tag%5B1%5D=${secondGiftCardTag}`,
```

Comment:
isn't there any way to make this url nicer in the test itself?

Reply:
TBH I did not want to do that since there is wip 🏗️  when it comes to filters - not sure how this is gonna look so did not want to spend time on it